src/mod/features/repel_fix.cpp

```diff
@@ -0,0 +1,56 @@
+#include "exlaunch.hpp"
+
+#include "externals/Dpr/EvScript/EvDataManager.h"
+#include "externals/ItemWork.h"
+#include "externals/PlayerWork.h"
+
+#include "logger/logger.h"
+
+const int32_t REPEL_ITEM_ID = 79;
+const int32_t SUPER_REPEL_ITEM_ID = 76;
+const int32_t MAX_REPEL_ITEM_ID = 77;
+
+const int32_t INFINITE_REPEL_FLAG = 2195;
```

Comment:
For flags, you can use the enums in FlagWork_Enums now instead of declaring a constant at the top now